tests/core/test_utils.py

```diff
@@ -1,14 +1,34 @@
+import pytest
+
 from winter.core.utils import cached_property
 
 
-def test_cached_property():
-    class _Test:
+class _Entity:
+
+    def __init__(self):
+        self._counter = 0
+
+    @cached_property
+    def method(self):
+        """Docstring"""
```

Comment:
What is it for?

Reply:
this check that if method has doc and after decorator also this property has doc. See cached_property.__init__